commands/stats.js

```diff
@@ -9,14 +9,19 @@ module.exports = {
     async execute(interaction) {
         const response = `
 __**global**__
-${await database.Player.count()} people have pinged at least once
-${await database.Player.sum('totalScore')} total pts have been gained
-${await database.Player.sum('score')} pts are currently owned
-${await database.Player.sum('clicks')} pings have been dealt with
+${await formatNumber(database.Player.count())} people have pinged at least once
+${await formatNumber(database.Player.sum('totalScore'))} total pts have been gained
+${await formatNumber(database.Player.sum('score'))} pts are currently owned
+${await formatNumber(database.Player.sum('clicks'))} pings have been dealt with
 __**personal**__
-${(await database.Player.findByPk(interaction.user.id)).clicks} total pings
-${(await database.Player.findByPk(interaction.user.id)).totalScore} total pts
+${(await formatNumber(database.Player.findByPk(interaction.user.id))).clicks} total pings
```

Comment:
see above comment, but also move .clicks in the database area
```suggestion
${(formatNumber(await database.Player.findByPk(interaction.user.id)).clicks)} total pings
```